doc/installationguide.md

```diff
@@ -1,208 +1,288 @@
 # Installation & Administration Guide
 
 ## Configuration
-The `activate()` function that starts the IoT Agent receives as single parameter with the configuration for the IoT Agent.
-The Agent Console reads the same configuration from the `config.js` file.
+The `activate()` function that starts the IoT Agent receives as single parameter 
```

Comment:
I'm afraid that some text formatting tool has changed the whole file and it's difficult to see the actual changes.

Maybe you could go back to the original version in master and apply only the new changes on it?

Reply:
To be honest, it was not only a change of formatting. The original content was not properly formatted.

Reply:
In that case I'd suggest to split the work in two pieces. In this PR, only changes related with multi-threading. In a next PR, only changes related with formatting (i.e. no new actual content).

Makes sense?

Reply:
Well, I would prefer that you see the 2 versions in parallel:
[Installation Guide](https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/installationguide.md)
[Forked Installation Guide](https://github.com/flopezag/iotagent-node-lib/blob/feature/cluster-support/doc/installationguide.md)

Reply:
Fixed in 547946e